app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,84 @@
+require 'pry'
+
+class TweetController < ApplicationController
+
+  get '/tweets' do
+
+    if !logged_in?
+      redirect to '/login'
+    end
+    @user = User.find_by_id(session[:user_id])
+    erb :'tweets/show'
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/new'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if (params[:tweet][:content]).empty?
+      redirect to '/tweets/new'
+    else
+      @user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.create(params[:tweet])
+      @user.tweets << @tweet
```

Comment:
for Lines 27-28 we should be using the associative build method in Active Record. 

```ruby 
@tweet = current_user.tweets.create(content: params[:content])
```